sass/_header.scss

```diff
@@ -0,0 +1,266 @@
+// COLOR VARIABLES
+$color-primary: #e5e5e5;
+$color-grey-dark-2: #28352f;
+
+// FONT VARIABLES
+$font-primary: 'Nunito', sans-serif;
+$font-display: 'Josefin Sans', sans-serif;
+
+// overall display setting
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%; // 10px/16px = 62.5% -> 1rem = 10px
+}
+
+// body {
+//   font-family: $font-primary;
+//   color: $color-grey-dark-2;
+//   font-weight: 300;
+//   line-height: 1.6;
+// }
+// // ---- START---- HEADER & LOGO STYLES
+// .Main-container {
+//   background-image: url(../img/background.jpg);
+//   background-repeat: no-repeat;
+//   display: flex;
+//   flex-direction: column;
+//   background-size: 100% 38%;
+//   height: 190vh;
+//   width: 400px;
+//   background-position: 0 80px;
+// }
+
+// .menu__link:hover {
+//   opacity: 0.6;
+// }
+
+// .container {
+//   display: flex;
+//   justify-content: space-between;
+//   padding: 1.8rem;
+// }
+
+// .Header {
+//   padding-bottom: 5rem;
+// }
+// // ---- START---- HEADLINES STYLE
+
+// .Headline {
+//   display: flex;
+//   flex-direction: column;
+// }
+
+// .Headline__primary {
+//   // border: 10px solid red;
+//   // padding: 4rem;
+//   // width: 32.9rem;
+//   width: 327px;
+// }
+
+// .Text__intro {
+//   font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+//   font-size: 3.5rem;
+//   font-weight: 1800;
+//   line-height: 4.2rem;
+//   text-align: center;
+//   text-emphasis-style: bolder;
+//   // padding: 2rem;
+// }
+
+// .headline__secondary {
+//   // border: 10px solid green;
+//   // width: 30%;
+//   // height: 164px;
+//   width: 327px;
+//   left: 24px;
+//   top: 314px;
+// }
+
+// .supporting_txt {
+//   // font-family:Inter;
+//   //styleName: Theme 2 / Body Reguler 1;
+//   font-family: $font-primary;
+//   font-size: 20px;
+//   font-weight: 400;
+//   line-height: 28px;
+//   letter-spacing: 0;
+//   text-align: center;
+//   padding: 2.5rem;
+// }
+
+// // h2{
+// //   font-family:$font-primary;
+// //   font-style: Bold;
+// //   font-size: 4rem;
+// //   line-height: 5.2rem;
+// //   line-height: 107%;
+// //   padding: 2.5rem;
+// // }
+// // ---- START---- SOCIAL MEDIA STYLE
+// .social_media {
+//   display: flex;
+//   justify-content: center;
+//   width: 280px;
+
+//   &__links {
+//     text-decoration: none;
+//   }
+
+//   &__links:hover {
+//     opacity: 0.6;
+//   }
+
+//   &__icons {
+//     height: 20px;
+//     left: 102.41666412353516px;
+//     // top: 536px;
+//     // justify-self: center;
+//     justify-self: center;
+//   }
+// }
+
+// .Social__media__list {
+//   list-style: none;
+//   margin-left: 1rem;
+// }
```

Comment:
Please remove all commented code from your codebase, it makes your codebase look untidy.